Dockerfile-default-cpu

```
@@ -20,8 +20,9 @@ ARG HOROVOD_PIP=horovod
 ARG HOROVOD_WITH_TENSORFLOW=1
 ARG HOROVOD_WITH_PYTORCH=1
 ARG HOROVOD_WITHOUT_MXNET=1
-ARG HOROVOD_WITHOUT_MPI=1
-ARG HOROVOD_CPU_OPERATIONS=GLOO
+ARG HOROVOD_WITH_MPI=${HOROVOD_WITH_MPI}
```

that should be equivalent to simply
```suggestion
ARG HOROVOD_WITH_MPI
```

---

I tried this but it didn't seem to work -- the values didn't appear correctly inside the Dockerfile which caused Horovod to not build MPI support.  Could be the version of Docker I'm using.  I'll try again to see if something else was wrong at the time.

---

I modified this to follow your suggestion and it built MPI correctly.  I must have missed one of the args in my original test that prevented Horovod from correctly using MPI. Thanks.